Documentation/gettingstarted/k8s-install-download-release.rst

```diff
@@ -1,12 +1,22 @@
-Download the Cilium release tarball and change to the kubernetes install directory:
-
-.. parsed-literal::
-
-    curl -LO |SCM_ARCHIVE_LINK|
-    tar xzvf |SCM_ARCHIVE_FILENAME|
-    cd |SCM_ARCHIVE_NAME|/install/kubernetes
-
 `Install Helm`_ to prepare generating the deployment artifacts based on the
 Helm templates.
 
 .. _Install Helm: https://helm.sh/docs/using_helm/#install-helm
+
+.. only:: stable
+
+   Setup helm repository:
+
+    .. code:: bash
+
+        helm repo add cilium http://releases.cilium.io/charts
```

Comment:
We're just waiting on a solution to the HTTP usage here.

Reply:
FWIW I've made advancements here, we now have HTTPS charts available here: https://cilium.github.io/charts .

The remaining item is to add a CNAME for `releases.cilium.io`, then I think we should be good to go.

Reply:
We reviewed the options, releases.cilium.io already has a few things there so we opted for https://helm.cilium.io instead. This page will need updating.

Reply:
Updated